build/docker/server/Dockerfile.el9

```diff
@@ -38,10 +30,21 @@ COPY pmm-client.tar.gz /tmp/
 RUN install -T -p -m 644 /opt/ansible/ansible.cfg /etc/ansible/ansible.cfg && \
     ansible-playbook -vvv -i 'localhost,' -c local /opt/ansible/pmm-docker/main.yml -e "pmm_running_in_docker=true" && \
     ansible-playbook -vvv -i 'localhost,' -c local /opt/ansible/pmm-docker/update.yml -e "pmm_running_in_docker=true" && \
-    ansible-playbook -vvv -i 'localhost,' -c local /opt/ansible/pmm/post-build-actions.yml -e "pmm_running_in_docker=true"
+    ansible-playbook -vvv -i 'localhost,' -c local /opt/ansible/pmm/post-build-actions.yml -e "pmm_running_in_docker=true" && \
```

Comment:
@talhabinrizwan FYI: I'm removing those `pmm_running_in_docker` vars as they don't seem to be used in v3.

Reply:
No, it's not used in v3 :)